silk-workspace/src/main/scala/org/silkframework/workspace/activity/workflow/LocalWorkflowExecutor.scala

```diff
@@ -179,6 +187,8 @@ case class LocalWorkflowExecutor(workflowTask: ProjectTask[Workflow],
             writeErrorOutput(operatorNode, executorOutput)
             process(result)
           }
+          stopwatch.stop(registry.timer(s"$prefix.workflow.operator.execution", "operator", operator.nodeId))
```

Comment:
Unfortunately, the node id is not unique between workflows so it's not clear, which workflow the resulting metric value belongs to. Is it possible to add two tags here? One for the workflow and another one for the operator?

Reply:
I added the **workflow (execution) ID** as a new tag pair.